tracex_project/extraction/tests/test_views.py

```diff
@@ -0,0 +1,390 @@
+"""Test cases for the views of the extraction app."""
+import json
+
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.test import TestCase, Client, RequestFactory
+from django.urls import reverse, resolve
+
+from extraction.forms import (
+    JourneyUploadForm,
+    JourneySelectForm,
+    FilterForm,
+    ResultForm
+)
+from extraction.models import (
+    PatientJourney,
+)
+from extraction.views import (
+    JourneyInputSelectView,
+    JourneyUploadView,
+    JourneySelectView,
+    JourneyDetailView,
+    JourneyFilterView,
+    ResultView,
+)
+
+
+class JourneyInputSelectViewTests(TestCase):
+    """Test cases for the JourneyInputSelectView."""
+
+    def setUp(self):  # pylint: disable=invalid-name
+        """Set up method that gets called everytime before tests are executed."""
```

Comment:
same as above. Maybe consider using another name/snake_case and adjust the docstring to be more declarative.

Reply:
Keep in mind that this implements a function that is inherited from the Django `TestCase` class. Django always calls this function automatically before tests are executed. If I were to rename this method, it would not get executed anymore.